### PR TITLE
chore/snmp profiles

### PR DESCRIPTION
**Pull request by thezackm**
overhauling existing docs to add steps for success that customers can follow in managing SNMP profiles for network monitoring

### CODE REVIEW

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (homelessbirds) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:22]:
``` suggestion
<Callout variant="important">
  New Relic does not provide support directly for the contents of the [kentik/snmp-profiles](https://github.com/kentik/snmp-profiles). If an existing SNMP profile is returning incorrect data or another breaking behavior, open an issue on the [kentik/snmp-profiles](https://github.com/kentik/snmp-profiles) repository. We are unable to provide timelines for modification or creation of SNMP profiles in the third party for `ktranslate`.
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2221]:
```suggestion
                        </Collapser>
                   </CollapserGroup>
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2557]:
```suggestion
          Putting all of these sections together, here's an example of the profile used for [APC UPS devices](https://github.com/kentik/snmp-profiles/blob/main/profiles/kentik_snmp/apc/apc_ups.yml):
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2685]:
```suggestion
    The creation of SNMP trap message profiles (also known as alerts, events, informs, or notifications) is quite simple. Generally it only requires you to investigate the vendor's MIB file to understand various trap message options.
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2822]:
```suggestion
          * **Notifications by an agent that implements this MIB module**: This is the definition of an actual trap message, identified by a unique OID.
```

Comment (rhetoric101) [reply src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2822]: Should `MIB` be capitalized as I did above?

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2823]:
```suggestion
            * [vmwVCNotifications](https://www.circitor.fr/Mibs/Html/V/VMWARE-VC-EVENT-MIB.php#vmwVCNotifications): This is the root notification OID which will cover the three defined types of messages available: [vpxdAlarm](https://www.circitor.fr/Mibs/Html/V/VMWARE-VC-EVENT-MIB.php#vpxdAlarm), [vpxdDiagnostic](https://www.circitor.fr/Mibs/Html/V/VMWARE-VC-EVENT-MIB.php#vpxdDiagnostic), and [vpxdAlarmInfo](https://www.circitor.fr/Mibs/Html/V/VMWARE-VC-EVENT-MIB.php#vpxdAlarmInfo).
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2824]:
```suggestion
          * **Notification parameters**: Commonly referred to as `variable bindings` or `varbinds`, these are the unique `key:value` pairs of alert data that the SNMP agent can possibly send as decorations on the trap messages.
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2825]:
```suggestion
            * In this MIB, there are eight possible varbinds&mdash;all referenced as the OIDs ranging from `1.3.6.1.4.1.6876.4.3.301` - `1.3.6.1.4.1.6876.4.3.308`. This will be unique to every vendor and MIB.
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2854]:
```suggestion
                         </Collapser>
                    </CollapserGroup>
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2927]:
```suggestion
To support full customization options so you can maintain your own collection settings and patterns, do the following: 
1. Clone the repository.
1. Make changes as you see fit.
1. Load your custom profile directory into your Docker container or Linux service on startup.
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2931]:
```suggestion
Assuming you are in a directory just above `snmp-profiles`, the command to add looks  like this:
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2958]:
```suggestion
  To remove the risk of inconsistent behavior in your environment, be sure to pass in your custom version of the profiles every time you launch a container or service. 
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2965]:
```suggestion
New Relic isn't involved in the process of reviewing profiles on the third-party repo. All requests for changes and updates must be addressed with Kentik's maintainers via Github issues.
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2967]:
```suggestion
Shortly after a profile is merged, new SNMP profiles are automatically available. You can pull the new version of the Docker image and launch a new container in your environment. For a Linux service, create a new release on the [kentik/ktranslate](https://github.com/kentik/ktranslate) repository.
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2971]:
```suggestion
In many cases SNMP offers a lot of data, but little provides actionable information. The [sample SNMP walk](#snmp-polling) above has 2,022 rows of results from OIDs, and the [resulting profile](https://github.com/kentik/snmp-profiles/blob/main/profiles/kentik_snmp/apc/apc_ups.yml) collects about 50 total OIDs. Following the basic patterns of **"Golden Signals"** defined by the [Site Reliability Engineering (SRE) discipline](https://sre.google/sre-book/monitoring-distributed-systems/); focus on collecting data that lets you know if there is anything that would stop the device from being able to perform whatever functions you expect it to perform. Building from that, you should collect measurements that tell you how well it is performing those functions.
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (rhetoric101) [inline src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2983]:
```suggestion
An OID table that lists all the running processes on a network appliance is an example of data that is available but not helpful. 

Collecting all running processes might sound useful, but since this is an appliance, you normally do not have the capability or the need to do anything with the processes that run inside it. Polling and storing tables with hundreds of items that you can't actually do anything with would not be efficient.
```

**Review by rhetoric101 — COMMENTED** [review]
@thezackm thanks again for adding this gigantic and helpful doc! I inserted some questions and suggestions. See what you think.

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (thezackm) [reply src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:17]: sounds great

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (thezackm) [reply src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:22]: thsi snippet came from legal, I wouldn't want to edit it any more

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (thezackm) [reply src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2967]: this isn't quite right. the process of creating a new release on the ktranslate repo is a manual process that one of the repo maintainers has to kick off. various customers/users will not be able to do this step.

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (thezackm) [reply src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2125]: i like this one; on `2127` should you add a space here?

```
*Create your own custom dashboard experience to view your data.
```

vs

```
* Create your own custom dashboard experience to view your data.
```

File: src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx
Comment (thezackm) [reply src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx:2147]: `SysObjectID` is the formal name of the attribute; `sysobjectid` is the setting in a YAML file. I'd defer to you on formatting (I'm not against it)

**Comment by rhetoric101** [comment]
@thezackm, for some reason, this PR isn't building successfully in the cloud. I am going to temporarily close this PR and then reopen it to trigger a fresh build.

**Review by rhetoric101 — APPROVED** [review]
@thezackm thanks again for all your hard work on this!

**Comment by rhetoric101** [comment]
@thezackm I am having trouble with the CLA job not reporting back, so I'm going to temporarily close this to see if reopening it will trigger the CLA job again.

**Pull request merged**
